src/main/java/Parser.java

```diff
@@ -0,0 +1,160 @@
+import java.util.Arrays;
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.FileWriter;
+
+public class Parser {
+
+    static Scanner scanner = new Scanner(System.in);
+    static final String LINEBREAK = "____________________________________________________________\n";
+    static ArrayList<String> commandList = new ArrayList<String>(
+            Arrays.asList("bye", "list", "done", "todo", "deadline", "event", "delete")
+    );
+    static final String EXITCOMMAND = "bye";
+    static final String TASKLIST_SAVE_PATH = "data/saveTaskList.txt";
+
+    public static void printLine() {
+        System.out.print(LINEBREAK);
+    }
+
+    static List list = new List();
+
+    public static void Instance() {
+        int saveNumber = 0;
+
+        //checking if there is a save file and accessing it
+        File savedList = new File(TASKLIST_SAVE_PATH);
+        boolean doesSaveExist = savedList.exists();
+        try {
+            if (!doesSaveExist) {
+                savedList.getParentFile().mkdirs();
+                savedList.createNewFile();
+            }
+        } catch (IOException e) {
+            System.out.println("☹ OOPS!!! I am unable to create a new directory or file :-(");
+        }
+        try {
+            Scanner saveFileScanner = new Scanner(savedList);
+
+            while (saveFileScanner.hasNext()) {
+                String currentRead = saveFileScanner.nextLine();
+                String[] taskSave = currentRead.trim().split(" \\| ");
+                switch (taskSave[0].trim()) {
+                    case "todo":
```

Comment:
maybe change "todo", "deadline","event" to constants in order to remove magic strings. In addition, maybe you should make the use of  Single Level of Abstraction Principle (SLAP) concept and for each case statement devise a method to handle the abstractions.